diff --git a/modules/contact/contact.module b/modules/contact/contact.module
index 2ba4c38..344863f 100644
--- a/modules/contact/contact.module
+++ b/modules/contact/contact.module
@@ -190,7 +190,7 @@ function contact_mail($key, &$message, $params) {
     case 'page_copy':
       $contact = $params['contact'];
       $message['subject'] .= t('[!category] !subject', array('!category' => $contact['category'], '!subject' => $params['subject']), $language->language);
-      $message['body'][] = t("!name sent a message using the contact form at !form.", array('!name' => $params['name'], '!form' => url($_GET['q'], array('absolute' => TRUE, 'language' => $language))), $language->language);
+      $message['body'][] = t("!name <@address> sent a message using the contact form at !form.", array('!name' => $params['name'], '@address' => $params['mail'], '!form' => url($_GET['q'], array('absolute' => TRUE, 'language' => $language))), $language->language);
       $message['body'][] = $params['message'];
       break;
     case 'page_autoreply':
