Index: color.module
===================================================================
--- color.module	(revision 4535)
+++ color.module	(working copy)
@@ -228,7 +228,7 @@
   // Preview
   $output .= drupal_render($form);
   $output .= '<h2>'. t('Preview') .'</h2>';
-  $output .= '<div id="preview"><div id="text"><h2>Lorem ipsum dolor</h2><p>Sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud <a href="#">exercitation ullamco</a> laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p></div><div id="img" style="background-image: url('. base_path() . $path . $info['preview_image'] .')"></div></div>';
+  $output .= '<div id="preview"><div id="link-fill"></div><div id="text"><h2>Lorem ipsum dolor</h2><p>Sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud <a href="#">exercitation ullamco</a> laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p></div><div id="img" style="background-image: url('. base_path() . $path . $info['preview_image'] .')"></div></div>';
 
   // Close wrapper
   $output .= '</div>';
Index: color.js
===================================================================
--- color.js	(revision 4535)
+++ color.js	(working copy)
@@ -62,6 +62,9 @@
     $('#text', form).css('color', inputs[4].value);
     $('#text a, #text h2', form).css('color', inputs[1].value);
 
+    // Link fill preview
+    $('#link-fill', form).css('backgroundColor', inputs[1].value);
+
     // Set up gradient
     var top = farb.unpack(inputs[2].value);
     var bottom = farb.unpack(inputs[3].value);
